<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlCenter</class>
 <widget class="QWidget" name="ControlCenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>325</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Control Center</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Settings</string>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLayout_2"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_latitude">
          <property name="text">
           <string>Latitude:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="latitude">
          <property name="inputMask">
           <string notr="true">00-00A</string>
          </property>
          <property name="maxLength">
           <number>6</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Longitude:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="longitude">
          <property name="inputMask">
           <string notr="true">000-00A</string>
          </property>
          <property name="maxLength">
           <number>7</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Date/Time (UTC):</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="datetime_utc">
          <property name="inputMask">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>MMSI:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Group:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="mmsi">
          <property name="inputMask">
           <string notr="true">000000000</string>
          </property>
          <property name="maxLength">
           <number>9</number>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="group">
          <property name="inputMask">
           <string notr="true">000000000</string>
          </property>
          <property name="maxLength">
           <number>9</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Working Channel:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QComboBox" name="working_channel">
          <property name="maxVisibleItems">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Designation:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QComboBox" name="designation"/>
        </item>
        <item row="8" column="1">
         <widget class="QCheckBox" name="receive_all">
          <property name="text">
           <string>Receive All Messages</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Received Message</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="received_message">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>DSC</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0,0,0,0,0">
            <item>
             <widget class="QPushButton" name="button_routine">
              <property name="text">
               <string>Routine</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_group">
              <property name="text">
               <string>Group</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_safety">
              <property name="text">
               <string>Safety</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_urgency">
              <property name="text">
               <string>Urgency</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_distress">
              <property name="text">
               <string>Distress</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_confirm">
              <property name="text">
               <string>Confirm</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_cancel">
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
